app/rankers/algorithms.py

```diff
@@ -16,6 +17,11 @@
 DEFAULT_ALPHA_PRIOR = 0.02
 DEFAULT_BETA_PRIOR = 1.0
 
+# These values were taken from: https://github.com/Pocket/feed-machine/blob/master/resources/models/b-0085-15k-age-05-syn-score-explore-locales-rr.json#L4-L5
+# They were experimentally derived in 2018: https://getpocket.atlassian.net/browse/P18-616
+DEFAULT_FIREFOX_BETA_PRIOR = 15600
```

Comment:
Agreed with your comment in the video: this looks low, but right now I'm not that worried about starting at the perfect value.